internal/app/table_failed.go

```diff
@@ -114,7 +115,12 @@ func (c *consoleWriter) prepareStyledPanic(
 		}
 		rows.WriteString(e.Output)
 	}
-	return lipgloss.JoinVertical(lipgloss.Left, styledPackageHeader, rows.String())
+	width, _, err := term.GetSize(int(os.Stdout.Fd()))
+	if err != nil {
+		width = 80
+	}
```

Comment:
96 is used for dividers. Should we use it here as well?

Reply:
Tried using the terminal width (with a fallback to 96) for both panic messages and dividers in 4ee4e5e2cdce944de990992a27d047eebdf5c60f